docs/source/developers/developers.rst

```diff
@@ -0,0 +1,53 @@
+Developers Guide
+================
+
+Adding WSGI services to an ESGF node
+------------------------------------
+
+It's easy to add new services developed with Flask, Django, etc. and proxy using the esgf-httpd configuration and mod_wsgi express.
+
+This guide assumes you have set up a project using flask at the example location ``/opt/esgf/flaskdemo/demo`` and you have the application entry point accessible in the ``/opt/esgf/flaskdemo/demo.wsgi``.  Your demo app must world-readable and recommended to be owned by the ``apache`` user and group.  These instructions assume to be run by a root user, as many server configs disallow a shell run under the apache user, but a regular user can be used to run the service for testing purposes.  
+
+- Create and activate a conda environment to run your webapp. 
+
+::
+
+    source /usr/local/conda/bin/activate
+    conda create -n flaskdemo
+    conda activate flaskdemo
+
+.. note:: 
+    if your app was developed in Python 2.7 you'll need to create the environment with the following instead:
+
+::
+
+    conda create -n flaskdemo
+
+- Install modules needed to run your app.  Our demo uses flask, but you could use django (and expect additional required packages.)  Note that diffculty has been encountered with mod_wsgi version 4.6.7, so we recommend an earlier version:  ::
+
+::
+
+    pip install flask 'mod_wsgi<4.6'
```

Comment:
Need to make the environment is activated

Reply:
done in previous step